src/kernel/Cargo.toml

```diff
@@ -23,13 +23,21 @@ linked_list_allocator = "0.10"
 tar-no-std = "0.2.0"
 object = { version = "0.32.1", default-features = false, features = ["read"] }
 addr2line = { version = "0.16.0", default-features = false, features = [
-    "rustc-demangle"
+    "rustc-demangle",
 ] }
 backtracer_core = { git = "https://github.com/twizzler-operating-system/backtracer", branch = "twizzler" }
 limine = "0.1.11"
 twizzler-queue-raw = { version = "*", path = "../lib/twizzler-queue-raw", default-features = false }
 syscall_encode = { version = "0.1.2" }
 volatile = "0.5"
+# for crypto
+p256 = { version = "0.13.2", default-features = false, features = ["ecdsa"] }
+sha2 = { version = "0.10.8", default-features = false, features = [
+    "force-soft",
+] }
+# [dev-dependencies] # kernel doesn't include dev dependencies when testing
```

Comment:
Can you open an issue for this? It should include them. We don't need to fix that to merge this, though.